builtin/mainmenu/settings/dlg_settings.lua

```diff
@@ -161,6 +161,11 @@ local function load()
 		note.requires = get_setting_info("enable_auto_exposure").requires
 		table.insert(content, idx, note)
 
+		idx = table.indexof(content, "enable_bloom") + 1
+		note = component_funcs.note(fgettext_ne("(The game will need to enable bloom as well)"))
```

Comment:
Why aren't these just part of the setting description btw.? You can't see it if you searched for the setting.

Reply:
> Why aren't these just part of the setting description btw.?

To make them more likely to be noticed

> You can't see it if you searched for the setting.

Hmm, that's indeed a problem (but not this PR's one)